app/controllers/gems_adoptions_controller.rb

```diff
@@ -0,0 +1,21 @@
+class GemsAdoptionsController < ApplicationController
+  
+  def index
+    @gems_adoptions = GemsAdoption.all
+  end
+
+  def create
+    @gems_adoption = GemsAdoption.new
+    @gems_adoption.gem = params[:name]
+    @gems_adoption.description = params[:info]
+    @gems_adoption.user_id = current_user.id
+    @gems_adoption.save
```

Comment:
What do you think about using the `GemsAdoption#create` method instead of setting each one of this things individually? Something like:

Also do you think we should handle the case when the GemsAdoption can not be persisted to the database, and let the user know something went wrong?